drivers/adxl345/include/adxl345_params.h

```diff
@@ -57,14 +54,12 @@ extern "C" {
 #define ADXL345_PARAM_SCALE_FACTOR (3.9)
 #endif
 #ifndef ADXL345_PARAMS
-#define ADXL345_PARAMS          { .i2c    = ADXL345_PARAM_I2C, \
-                                  .addr   = ADXL345_PARAM_ADDR, \
-                                  .rate   = ADXL345_PARAM_RATE, \
-                                  .range  = ADXL345_PARAM_RANGE, \
-                                  .offset = ADXL345_PARAM_OFFSET, \
-                                  .interrupt = ADXL345_PARAM_INTERRUPT, \
-                                  .full_res = ADXL345_PARAM_FULL_RES, \
-                                  .scale_factor = ADXL345_PARAM_SCALE_FACTOR}
+#define ADXL345_PARAMS          { .i2c    = ADXL345_PARAM_I2C,   \
+                                  .addr   = ADXL345_PARAM_ADDR,  \
+                                  .offset = ADXL345_PARAM_OFFSET }
```

Comment:
Because I copy/paste this  from another driver without even thinking ! I'll change it.